readme.md

```diff
@@ -319,6 +319,7 @@ Thanks for contributing! 🦋🙌
 
 ### Fixes for GitHub shortcomings
 
+- [](# "fix-first-tab-length") [The first tab space in file diff appears 2 spaces shorter than it should be.](https://user-images.githubusercontent.com/37769974/86880618-91e8ac80-c10a-11ea-8679-e7b61b253f54.png)
```

Comment:
This looks good, but can you add a red arrow (any style) pointing to the last 2-space tab in the before state? This would make it super clear about what the problem is